File: Java/백준/Bronze/31636. 三連続 （Three Consecutive）/三連続 （Three Consecutive）.java
import java.util.Scanner;

public class Main {
    public static void main (String[] args) {
        Scanner sc = new Scanner (System.in);
        
        int n = sc.nextInt();
        String s = sc.next();
        
        boolean found = false;
        
        for (int i = 0; i <= n - 3; i++) {
            if (s.charAt(i) == 'o' && s.charAt(i+1) == 'o' && s.charAt(i+2) == 'o') {
                found = true;
                break;
            }
        }
        
        if (found) {
            System.out.println("Yes");
        } else {
            System.out.println("No");
        }
        
        sc.close();
    }
}

File: C++17/백준/Bronze/31636. 三連続 （Three Consecutive）/三連続 （Three Consecutive）.cc
#include <iostream>
#include <string>
using namespace std;

int main() {
    int n;
    cin >> n;
    
    string s;
    cin >> s;
    
    bool found = false;
    
    for (int i = 0; i + 2 < n; i++) {
        if (s[i] == 'o' && s[i + 1] == 'o' && s[i + 2] == 'o') {
            found = true;
            break;
        }
    }
    
    if (found) {
        cout << "Yes";
    } else {
        cout << "No";
    }
    
    return 0;
}